sql/core/src/main/scala/org/apache/spark/sql/streaming/DataStreamWriter.scala

```diff
@@ -379,7 +379,7 @@ final class DataStreamWriter[T] private[sql](ds: Dataset[T]) {
 
   private var outputMode: OutputMode = OutputMode.Append
 
-  private var trigger: Trigger = Trigger.ProcessingTime(0L)
+  private var trigger: Trigger = Trigger.processingTime(0L)
```

Comment:
OK @ConeyLiu . We can't make this change in any event. If you can get the rule I added working, great, otherwise, just comment on it and ignore it.